app/uk/gov/hmrc/trusts/controllers/OtherIndividualTransformationController.scala

```diff
@@ -21,49 +21,70 @@ import org.slf4j.LoggerFactory
-    implicit request => {
-      request.body.validate[RemoveOtherIndividual] match {
-        case JsSuccess(otherIndividual, _) =>
-          transformService.removeOtherIndividual(utr, request.identifier, otherIndividual) map { _ =>
-            Ok
-          }
-        case JsError(_) => Future.successful(BadRequest)
+    def removeOtherIndividual(utr: String): Action[JsValue] = identify.async(parse.json) {
+      implicit request => {
+        request.body.validate[RemoveOtherIndividual] match {
+          case JsSuccess(otherIndividual, _) =>
+            transformService.removeOtherIndividual(utr, request.identifier, otherIndividual) map { _ =>
+              Ok
+            }
+          case JsError(_) => Future.successful(BadRequest)
+        }
+      }
+    }
+
+    def amendOtherIndividual(utr: String, index: Int): Action[JsValue] = identify.async(parse.json) {
+      implicit request => {
+        request.body.validate[NaturalPersonType] match {
+          case JsSuccess(otherIndividual, _) =>
+            transformService.amendOtherIndividualTransformer(
+              utr,
+              index,
+              request.identifier,
+              otherIndividual
+            ) map { _ =>
+              Ok
+            }
+          case JsError(errors) =>
+            logger.warn(s"[OtherIndividualTransformationController][addOtherIndividualTransformer] " +
+              s"Supplied json could not be read as an Unidentified Beneficiary - $errors")
+            Future.successful(BadRequest)
+        }
       }
     }
-  }
 
-  def amendOtherIndividual(utr: String, index: Int): Action[JsValue] = identify.async(parse.json) {
-    implicit request => {
-      request.body.validate[NaturalPersonType] match {
-        case JsSuccess(otherIndividual, _) =>
-          transformService.amendOtherIndividualTransformer(
-            utr,
-            index,
-            request.identifier,
-            otherIndividual
-          ) map { _ =>
-            Ok
-          }
-        case JsError(errors) =>
-          logger.warn(s"[OtherIndividualTransformationController][amendOtherIndividual]" +
-            s" Supplied json could not be read as an Other Individual - $errors")
-          Future.successful(BadRequest)
+    def addOtherIndividual(utr: String): Action[JsValue] = identify.async(parse.json) {
+      implicit request => {
+        request.body.validate[DisplayTrustNaturalPersonType] match {
+          case JsSuccess(otherIndividual, _) =>
+
+            transformService.addOtherIndividualTransformer(
+              utr,
+              request.identifier,
+              otherIndividual
+            ) map { _ =>
+              Ok
+            }
+          case JsError(errors) =>
+            logger.warn(s"[OtherIndividualTransformationController][addOtherIndividualTransformer] " +
+              s"Supplied json could not be read as an Unidentified Beneficiary - $errors")
```

Comment:
This log should not be unidentified beneficiary

Reply:
see above